docs/generated/errors-list.md

```diff
@@ -237,8 +237,12 @@ The `galasactl` tool can generate the following errors:
 - GAL1238E: Failed to get streams. Unexpected http status code {} received from the server. Error details from the server are not in a valid json format. Cause: '{}'
 - GAL1239E: Failed to get streams. Unexpected http status code {} received from the server. Error details from the server are: '{}'
 - GAL1240E: Failed to get streams. Unexpected http status code {} received from the server. Error details from the server are not in the json format.
-- GAL1241E: The test stream could not be deleted by name because it was not found by the Galasa service. Try listing streams using 'galasactl streams get' to identify the one you wish to delete
-- GAL1242E: Failed to test stream from database by stream name.
+- GAL12412E: Failed to delete test stream with the given name from the Galasa service
+- GAL1242E: Failed to delete streams. Unexpected http status code {} received from the server.
+- GAL1243E: Failed to delete streams. Unexpected http status code {} received from the server. Error details from the server could not be read. Cause: {}
+- GAL1244E: Failed to delete streams. Unexpected http status code {} received from the server. Error details from the server are not in a valid json format. Cause: '{}'
+- GAL1245E: Failed to delete streams. Unexpected http status code {} received from the server. Error details from the server are: '{}'
+- GAL1246E: Failed to delete streams. Unexpected http status code {} received from the server. Error details from the server are not in the json format.
```

Comment:
"Failed to delete test stream '{}'" instead of "Failed to delete streams" (since we're only deleting one stream)?